services/streammanager/kafka/kafkamanager.go

```diff
@@ -63,6 +63,18 @@ func loadCertificate() {
 	}
 }
 
+func getDefaultConfiguration() *sarama.Config {
+	config := sarama.NewConfig()
+	config.Net.DialTimeout = time.Duration(kafkaDialTimeoutInSec) * time.Second
+	config.Net.WriteTimeout = time.Duration(kafkaWriteTimeoutInSec) * time.Second
+	config.Net.ReadTimeout = time.Duration(kafkaWriteTimeoutInSec) * time.Second
```

Comment:
We are using the same `kafkaWriteTimeoutInSec` for both and read and write timeouts. Is that on purpose?

Reply:
Yes. Basically, we only publish and dont consume. It is for how long the client should wait for getting the response.
                 ```
// All three of the below configurations are similar to the
		// `socket.timeout.ms` setting in JVM kafka. All of them default
		// to 30 seconds.
		DialTimeout  time.Duration // How long to wait for the initial connection.
		ReadTimeout  time.Duration // How long to wait for a response.
		WriteTimeout time.Duration // How long to wait for a transmit.
```